### PR TITLE
remove additional redundant "press any key to exit"

### PR DESCRIPTION
**Pull request by lovettchris**
fix: remove additional redundant "press any key to exit" that was happening only on windows.
fix: remove reference to "Lean 3" in NoDefaultToolchain error message.

The reason for this "where installation happens in a console that may have opened just for this purpose" is no longer valid.  The latest vscode extension when you install elan, you now get "press any key to continue" twice, which is odd, so this fixes that making the vscode extension experience consistent across platforms.

### CODE REVIEW

File: src/elan/errors.rs [hunk 1]
```diff
@@ -24,7 +24,7 @@ error_chain! {
             display("toolchain '{}' is not installed", t)
         }
         NoDefaultToolchain {
-            description("no default toolchain configured. run `elan default stable` to install & configure the latest Lean 3 community release.")
+            description("no default toolchain configured. run `elan default stable` to install & configure the latest Lean community release.")
```
[the remark this answers is not in the document]
Comment (lovettchris) [reply src/elan/errors.rs:27]: right but it is weird getting this message from the lean4 vscode extension, can we add a command line argument to elan or something that says we are only interested in lean4 so we don't get this confusing message?

File: README.md [hunk 2]
```diff
@@ -77,3 +77,33 @@ On some systems, `lean`/`leanpkg` will not work out of the box even if installed
 * Removed Windows installer... for now?
 * Merged `CARGO_HOME` and `RUSTUP_HOME`
 * Removed options to configure host triple
+
+# Build
```
Comment (Kha) [inline README.md:81]: Can you adjust these instructions to https://github.com/leanprover/elan/issues/44#issuecomment-939251683 (env vars shouldn't be necessary anymore, acquiring openssl on Windows might be easier via `vcpkg`).

File: src/elan/errors.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (lovettchris) [reply src/elan/errors.rs:27]: I just don't see why you want to have to update elan source code every time a new version of lean comes out.  What if this was instead a call to query the current github releases list?

File: README.md (hunk 2 above)
Comment (lovettchris) [reply README.md:80]: Awesome, thanks for fixing that.  I updated the readme.

File: src/elan/errors.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (Kha) [reply src/elan/errors.rs:27]: Yes, please put it back for now so we can get this PR merged :) . It doesn't look like there will be a community release of Lean 4 currently, so this describes a special circumstance anyway.

[the remark this answers is not in the document]
Comment (lovettchris) [reply src/elan/errors.rs]: Sure thing, it is done.

**Comment by lovettchris** [comment]
> A bit late, but I hope this doesn't break/affect the Lean 3 extension's Lean installation? /cc @gebner

I just tested bootstrapping of lean3 from the vscode lean3 extension and it was able to download and install lean just fine, so looks like it still works.

**Comment by gebner** [comment]
> I just tested bootstrapping of lean3 from the vscode lean3 extension and it was able to download and install lean just fine, so looks like it still works.

From what I can tell, the change isn't released yet, right?

Either way, the change looks innocent enough and I don't expect it to break anything.  To be on the safe side we could post a heads-up on zulip once the release is out.

**Comment by lovettchris** [comment]
> > I just tested bootstrapping of lean3 from the vscode lean3 extension and it was able to download and install lean just fine, so looks like it still works.
> 
> From what I can tell, the change isn't released yet, right?
> 
> Either way, the change looks innocent enough and I don't expect it to break anything. To be on the safe side we could post a heads-up on zulip once the release is out.

Where is the lean3 vscode extension source code?  I can try it in the debugger and swap out elan.exe after it unzips the archive then see what happens....

**Comment by Kha** [comment]
It's at https://github.com/leanprover/vscode-lean. But I made testing easier by going ahead and releasing the change :) .

**Pull request merged**